src/style.css

```diff
@@ -0,0 +1,109 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+  color: #3d4e61;
+}
+
+.html {
+  font-family: Avenir, sans-serif;
+}
+
+.input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  font-size: 16px;
+}
+
+.inputSmall::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  font-size: 14px;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  box-sizing: border-box;
```

Comment:
move this to `*`

Reply:
Don't understand this one. It doesn't work this way. Searchbars becoming wider than screen and go out of the rigth side of screen.

Reply:
should be 
```
* {
  box-sizing: border-box;
}
```